alns/select/RandomSelect.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+from alns.select.OperatorSelectionScheme import OperatorSelectionScheme
+
+
+class RandomSelect(OperatorSelectionScheme):
+    """
+    Randomly selects operator pairs with uniform probability. The operator
+    pairs respect the operator coupling matrix.
+    """
+
+    def __call__(self, rnd, best, curr):
+        """
+        Selects a destroy operator with uniform probability. Then selects a
+        repair operator with uniform probability, respecting the operator
+        coupling matrix.
+        """
+        p = np.sum(self.op_coupling, axis=1) / np.sum(self.op_coupling)
+        d_idx = rnd.choice(np.arange(self.num_destroy), p=p)
```

Comment:
The docstring describes something different than the implementation :-). Do we want to sample each combination (d_op, r_op) uniformly or do we want to sample d_op and r_op separately?

Reply:
I think the current implementation makes sense. When there's no op coupling then the operators are all sampled uniformly and independently. If there is op coupling, then we should look at the operator pairs instead and sample those uniformly.

So changing the docstring works for me.

Reply:
Suggestion: maybe we can make this even easier? Find all non-zero (i, j) in the op_coupling matrix. Then apply rnd.choice on those index tuples (i, j)?